lib/linkedin-scraper/profile.rb

```diff
@@ -155,6 +155,24 @@ def recommended_visitors
       end
     end
 
+    def projects
+      @projects ||= @page.search('.background-projects/div').map do |project|
+        project = project.at('div')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.